### PR TITLE
Feature add nim

### PR DESCRIPTION
**Pull request by ynfle**
Hey! 👋 
Saw your wonderful post on dev.to and @juancarlospaco's comment about Nim and decided submit a PR and see if you'd accept it. I have no idea how you test it & if you wanted me to add the `run.sh` script to the folder, so let me know

### CODE REVIEW

File: nim/nim.cfg [hunk 1]
```diff
@@ -0,0 +1 @@
+-d:release
```
Comment (juancarlospaco) [inline nim/nim.cfg:1]:
```suggestion
-d:danger -d:lto -d:strip --threads:on
```

File: nim/src/views/general.nim [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#? stdtmpl(subsChar = '$', metaChar = '#')
```
Comment (juancarlospaco) [inline nim/src/views/general.nim:1]:
```suggestion
#? stdtmpl
```

File: nim/src/views/general.nim [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#
+#proc renderMain*(ip: string): string =
+#  result = ""
```
Comment (juancarlospaco) [inline nim/src/views/general.nim:4]:
```suggestion
```

File: nim/src/views/general.nim (hunk 2 above)
Comment (ynfle) [reply nim/src/views/general.nim:1]: This is what was done in Nim in Action, so I assumed it was standard

File: nim/src/views/general.nim [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+<html>
+    <head>
+    </head>
+    <body>
+        <h1>Your ip address is: ${ip}</h1>
```
Comment (juancarlospaco) [inline nim/src/views/general.nim:9]:
```suggestion
        <h1>Your ip address is: $ip </h1>
```

File: nim/src/views/general.nim (hunk 3 above)
Comment (ynfle) [reply nim/src/views/general.nim:4]: https://github.com/wickdChromosome/simple_webapp_comparison/pull/2#discussion_r514645096

File: nim/src/nim.nim [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import jester
+import views/general
```
Comment (juancarlospaco) [inline nim/src/nim.nim:2]:
```suggestion
import jester, views/general
```

Comment (ynfle) [reply nim/src/nim.nim:2]: I would've done the other way because it separates the library imports versus the local ones

**Comment by wickdChromosome** [comment]
Awesome, thanks! :rocket:

**Comment by juancarlospaco** [comment]
Maybe it can be a "Part 2" of the post hahaha. :smile: :+1:

**Comment by ynfle** [comment]
@wickdChromosome, if you find this PR useful and wouldn't mind I'd appreciate an hacktoberfest-accepted label.
My contribution isn't contingent on that.
Thanks

**Pull request merged**